src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -117,73 +139,154 @@ public JsonWebKeySet(string json, JsonSerializerSettings jsonSerializerSettings)
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = JsonWebKeySetParameterNames.Keys, Required = Required.Default)]
         public IList<JsonWebKey> Keys { get; private set; } = new List<JsonWebKey>();
 
+        /// <summary>
+        /// Flag that controls whether unresolved JsonWebKeys will be included in the resulting collection of <see cref="GetSigningKeys"/> method.
+        /// </summary>
+        [DefaultValue(true)]
+        public static bool SkipUnresolvedJsonWebKeys { get; set; } = true;
```

Comment:
Can we add the pattern of a Default static then an instance variable that is set to the default. Similar to here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/ce23b8c4002d168d6ab2f9d971a9993e5cba99cc/src/Microsoft.IdentityModel.Tokens/TokenHandler.cs#L47

Reply:
I'll make this change to maintain consistency.

Reply:
addressed with f27d3344907beb84e704cac8674543090a41b68f